microsoft-edge/web-platform/password-reveal.md

```diff
@@ -5,13 +5,13 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 02/23/2022
+ms.date: 08/19/2022

```

Comment:
This md file doesn't need to be changed. The only change you made in the content is adding a space at the end of line 14. This is minor, but I suggest removing this extra space and reverting the ms.date to what it originally was.